backend-api/src/functions/asyncCredential/asyncCredentialHandler.test.ts

```diff
@@ -829,114 +894,32 @@ describe("Async Credential", () => {
               govuk_signin_journey_id: "mockGovukSigninJourneyId",
             }),
           });
+          const mockEventService = new MockEventWriterSuccess();
+          dependencies.eventService = () => mockEventService;
           dependencies.sessionService = () =>
-            new MockSessionServiceCreateSessionErrorResult(
-              env.SESSION_TABLE_NAME,
-            );
+            new MockSessionServiceCreateSuccessResult();
 
           const result = await lambdaHandlerConstructor(dependencies, event);
 
+          expect(mockEventService.auditEvents[0]).toEqual(
+            "DCMAW_ASYNC_CRI_START",
+          );
           expect(mockLogger.getLogMessages()[0].logMessage.message).toBe(
-            "ERROR_CREATING_SESSION",
+            "COMPLETED",
```

Comment:
I am aware that we don't have a STARTED log. I made the decision to not change this in this PR because I did not want to update every single unit test.